scalyr_agent/util.py

```diff
@@ -1488,6 +1571,75 @@ def _initialize_token_queue(self):
         self._ripe_time = self._get_next_ripe_time()
         self._token_queue_cv.release()
 
+    
+    def _increase_cluster_rate(self):
```

Comment:
This method and corresponding _decrease_cluster_rate are the key logic to review.